model/host/db.go

```diff
@@ -756,11 +756,10 @@ func MarkStaleBuildingAsFailed(ctx context.Context, distroID string) error {
 	spawnedByTaskKey := bsonutil.GetDottedKeyName(SpawnOptionsKey, SpawnOptionsSpawnedByTaskKey)
 	query := bson.M{
 		distroIDKey:      distroID,
-		UserHostKey:      false,
 		spawnedByTaskKey: bson.M{"$ne": true},
 		ProviderKey:      bson.M{"$in": evergreen.ProviderSpawnable},
 		StatusKey:        evergreen.HostBuilding,
-		CreateTimeKey:    bson.M{"$lt": time.Now().Add(-15 * time.Minute)},
```

Comment:
Why are we lowering it to 5 minutes from 15?

Reply:
I reverted it back to 15. I only lowered it for the sake of testing a bit faster.